src/Components/App/App.js

```diff
@@ -24,9 +24,14 @@ const App = () => {
 
   const get4RdmPokemon = () => {
     const pokemons = [{}, {}, {}, {}]
-    return pokemons.map(emptySlot => {
-      const randomIndex = Math.floor(Math.random() * allPokemon.length)
-      return allPokemon[randomIndex]
+    let randomIndexes = []
```

Comment:
Refactor to not have duplicate pokemon displaying to button choices

Reply:
Nice attention to detail!

Reply:
Good Idea to implement the bug fix!